### PR TITLE
Feature/automating android

### PR DESCRIPTION
**Pull request by sayaansaha**
1. updated settings.gradle to include :AndroidSDK
2. updated AndroidManifest.xml corrected <intent-filter> to properly add app to App Launcher
3. Created a Load test with button

### CODE REVIEW

File: .idea/gradle.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/app" />
+            <option value="$PROJECT_DIR$/../Leanplum-Android-SDK/AndroidSDK" />
```
Comment (benmarten) [inline .idea/gradle.xml:13]: can u please add the UIEditor in the same manner?

File: app/build.gradle [hunk 2]
```diff
@@ -44,7 +51,7 @@ android {
     }
 }
 
-def LP_VERSION = System.getenv('VERSION') ?: '2.2.2';
+def LP_VERSION = System.getenv('VERSION') ?: '2.2.3-SNAPSHOT.167';
```
Comment (benmarten) [inline app/build.gradle]: let's point this to the latest stable version for now ;) 2.2.2

File: app/build.gradle [hunk 3]
```diff
@@ -58,7 +65,11 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:25.3.1'
     testCompile 'junit:junit:4.12'
 
-    compile 'com.leanplum:Leanplum:' + LP_VERSION
-    compile 'com.leanplum:UIEditor:' + LP_VERSION
+    packagedCompile 'com.leanplum:Leanplum:' + LP_VERSION
+    packagedCompile 'com.leanplum:UIEditor:' + LP_VERSION
+
+    sourcesDebugCompile project(path: ':AndroidSDK', configuration: 'debug')
+    sourcesReleaseCompile project(path: ':AndroidSDK', configuration: 'release')
```
Comment (benmarten) [inline app/build.gradle]: add uieditor here please

File: app/src/main/java/com/leanplum/android/example/Configure.java [hunk 4]
```diff
@@ -15,13 +15,13 @@
...
 class Configure {
-  static final String APP_ID = "app_BWTRIgOs0OoevDfSsBtabRiGffu5wOFU3mkxIxA7NBs";
-  static final String PRODUCTION_KEY = "prod_A1c7DfHO6XTo2BRwzhkkXKFJ6oaPtoMnRA9xpPSlx74";
-  static final String DEVELOPMENT_KEY = "dev_Bx8i3Bbz1OJBTBAu63NIifr3UwWqUBU5OhHtywo58RY";
+  static final String APP_ID = "app_LzkBKz08zXAFxKCrSyeScGHLlFX6XUVQBdT5za03Wks";
+  static final String PRODUCTION_KEY = "prod_B32xfU8yOurat0vtBN6MUyHKS6emTkNRFeZpfq7l3mg";
+  static final String DEVELOPMENT_KEY = "dev_GyoNGztMiVHIdpDvq9ilrL6jcs6n4IcCpp8oJvfbhj0";
 
-  static final String API_HOST_NAME = "10.0.2.2:3000";
+  static final String API_HOST_NAME = "leanplum-qa-1372.appspot.com";
```
Comment (benmarten) [inline app/src/main/java/com/leanplum/android/example/Configure.java]: lets' actually point this to the prod api host name, `api.leanplum.com`

File: app/src/main/java/com/leanplum/android/example/Configure.java [hunk 5]
```diff
@@ -15,13 +15,13 @@
...
   static final Boolean API_SSL = false;
 
-  static final String SOCKET_HOST_NAME = "dev.leanplum.com";
+  static final String SOCKET_HOST_NAME = "dev-qa.leanplum.com";
```
Comment (benmarten) [inline app/src/main/java/com/leanplum/android/example/Configure.java]: revert

File: app/src/main/java/com/leanplum/android/example/fragments/PushNotificationsFragment.java [hunk 6]
```diff
@@ -39,7 +40,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
       public void onClick(View v) {
+        Leanplum.track("Example");
+      }
+    });
 
+    Button loadTest = (Button) root.findViewById(R.id.load_test);
+    loadTest.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        for(int i=0; i < 10002; i++) {
```
Comment (benmarten) [inline app/src/main/java/com/leanplum/android/example/fragments/PushNotificationsFragment.java]: can you press code auto format in android studio please?

File: app/src/main/java/com/leanplum/android/example/fragments/PushNotificationsFragment.java [hunk 7]
```diff
@@ -39,7 +40,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+          Leanplum.track("Load TestF");
```
Comment (benmarten) [inline app/src/main/java/com/leanplum/android/example/fragments/PushNotificationsFragment.java]: improve the string a little bit, how about `"load_test_iteration" + i`

File: app/src/main/res/layout/show_example_layout.xml [hunk 8]
```diff
@@ -10,5 +10,12 @@
         app:layout_constraintBottom_toBottomOf="parent" app:layout_constraintHorizontal_bias="0.502"
         app:layout_constraintLeft_toLeftOf="parent" app:layout_constraintRight_toRightOf="parent"
         tools:layout_editor_absoluteY="454dp" />
+    <Button
```
Comment (benmarten) [inline app/src/main/res/layout/show_example_layout.xml:13]: can we align/center the button nicely please?

File: settings.gradle [hunk 9]
```diff
@@ -1 +1,3 @@
 include ':app'
```
Comment (benmarten) [inline settings.gradle:1]: add uieditor as well please

**Review by benmarten — CHANGES_REQUESTED** [review]
thanks for your changes. If you could take a moment to implement the changes it would be awesome! Thanks :)

**Comment by benmarten** [comment]
please limit to 1/2 reviewers. generally only one reviewer is required. if its a big or critical change, two reviewers are ok too...

File: app/build.gradle (hunk 2 above)
Comment (sayaansaha) [reply app/build.gradle]: of course!

**Review by benmarten — APPROVED** [review]
awesome! thx!!!!

**Comment by benmarten** [comment]
if you get a chance, can you align the button nicely please? ;)

**Pull request merged**